src/app/components/common/common.component.ts

```diff
@@ -9,24 +10,37 @@ import { ContentAPIService } from '../../services/content.api.service';
 
 })
 
+@Injectable()
 export class CommonComponent implements OnInit {
 
-  protected stContent: string[] = [];
+  protected stContent: String[] = [];
   protected rtrnContent: String[][];
-  constructor(protected http: HttpClient, protected contentSrv: ContentAPIService, protected passedContent: string[]) {
-    this.stContent = passedContent;
-    console.log(passedContent);
+  constructor(protected http: HttpClient, 
+    protected contentSrv: ContentAPIService,
+    //protected passedContent: String[]
+    ) {
+    // this.stContent = passedContent;
+    // console.log(passedContent);
   }
 
   ngOnInit() {
     // Get Externalized Content
```

Comment:
so... ngOnInit basically is empty. @jmilleriec  we should just delete?

Reply:
we can, the previous challenge was being caused by dependency injection, constructor and ngOnInit, so I took everything out and simply required method invocation on ngOnInit in the inherited class